app/utils/markdown.js

```diff
@@ -25,34 +32,52 @@ export const getMarkdownTextStyles = makeStyleSheetFromTheme((theme) => {
             color: theme.linkColor
         },
         heading1: {
-            fontSize: 17,
+            fontSize: normalizeFontSizeByDevice(16),
             fontWeight: '700',
-            lineHeight: 25
+            lineHeight: normalizeFontSizeByDevice(22)
+        },
+        heading1Text: {
+            paddingBottom: normalizeFontSizeByDevice(6)
```

Comment:
Just to confirm, this has to be on the Text and can't be on the View wrapping it, right?

Reply:
Yes. Does nothing on the View wrapper